src/pybind/mgr/telemetry/module.py

```diff
@@ -854,6 +1017,27 @@ def show(self, channels: Optional[List[str]] = None) -> Tuple[int, str, str]:
         Show report of all channels
         '''
         report = self.get_report(channels=channels)
+
+        # Formatting the perf histograms so they are human-readable. This will change the
+        # ranges and values, which are currently in list form, into strings so that
+        # they are displayed horizontally instead of vertically.
+        try:
+            for config in report['osd_perf_histograms']:
+                for histogram in config:
+                    # Adjust ranges by converting lists into strings
+                    for axis in config[histogram]['axes']:
+                        for i in range(0, len(axis['ranges'])):
+                            axis['ranges'][i] = str(axis['ranges'][i])
+                    # Adjust values by converting lists into strings
+                    for i in range(0, len(config[histogram]['values'])):
+                        config[histogram]['values'][i] = str(config[histogram]['values'][i])
```

Comment:
Had to modify these lines a bit to accommodate returning a list of configs. The logic hasn't changed.